edgelet/version.txt

```diff
@@ -1 +1 @@
-1.1.0~dev
+1.2.0_rc4
```

Comment:
I'll update this on https://github.com/Azure/iotedge/pull/4512 this PR as part of the release process.